converters/google/vendor_utils.go

```diff
@@ -62,5 +62,5 @@ func getProjectFromSchema(projectSchemaField string, d converter.TerraformResour
 	if config.Project != "" {
 		return config.Project, nil
 	}
-	return "", fmt.Errorf("required field '%s' is not set", projectSchemaField)
+	return "", fmt.Errorf("required field '%s' is not set, you may use --project=my-project to provide default project to resolve the issue", projectSchemaField)
```

Comment:
could this be: provide _a_ default project

Reply:
so something simply like 
```suggestion
	return "", fmt.Errorf("required field '%s' is not set, provide a default project", projectSchemaField)
```

Reply:
no, the version you had before was great, it was just missing the word "a" between `provide` and `default project`. So the full message would be:

```
required field '%s' is not set, you may use --project=my-project to provide a default project to resolve the issue
```

Reply:
oh okay, sorry I got that wrong. This is clear now, I will push the changes. Thanks

Reply:
No worries! Thanks for making the update; I'm looking forward to incorporating this change!

Reply:
You're welcome. Updated the changes.